help/de/Help_ch_GUIMenus.xml

```diff
@@ -3248,6 +3401,32 @@
         </menuchoice>
         ein- und ausgeblendet werden.
       </para>
+
+    <para>Möchten
+      Sie die Darstellung in der <guilabel>Zusammenfassungsleiste</guilabel> umstellen, so können Sie das mit Betätigen der Schaltfläche mit dem
+      <guilabel>kleinen nach unten zeigende Pfeil</guilabel> auf der rechten Seite des
+      Hauptkontofensters (direkt neben dem <guilabel>Rollbalken</guilabel>) machen. Es öffnet sich
+      eine Auswahlliste mit unterschiedlichen Optionen. Dies Auswahl ist nur für
+      die aktuell geöffenten Ansicht der Kontenstruktur gültig.
+    </para>
+
+    <figure pgwide="1">
+      <title>Die Zusammenfassungsleiste</title><screenshot>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="figures/summarybar-selector.png"
+                   srccredit="Christian Wehling" />
+          </imageobject>
+        </mediaobject>
+      </screenshot>
+    </figure>
+
```

Comment:
Staatsanwalt ermittelt wegen des Anfangsverdachts der Konkurs-Verschleppung!

Kannst du das nochmal mit positivem Eigenkapital aufnehmen?
Oder eine Geschichte dazu erzählen, aber das paßt eher in den Guide: In diesem Fall sollten ihre Alarmglocken schrillen, da Se offenbar pleite sind…

Reply:
Jetzt habe ich einen negativen Gewinn, ist das für eine Hilfedokumentation besser?

Reply:
Hast du irgendwo sowas wie Vorzeichenumkehr falsch eingestellt oder einen Vorzeichendreher eingegeben?

Ein schönes Normalfall-Beispiel hätte etwa z.B. EK1000, Ertrag 100 , von mir aus auch eine Null mehr.